test/org/loklak/harvester/YoutubeScraperTest.java

```diff
@@ -52,13 +53,17 @@ public void parseFromInputStreamTest() throws IOException {
     public void parseFromBufferedReaderTest() throws IOException {
         YoutubeScraper ytubeScrape = new YoutubeScraper();
         String url = "https://www.youtube.com/watch?v=KVGRN7Z7T1A";
+        String postType = "video";
+        String postScraper = "youtube";
 
         try {
             //Check Network issue
             BufferedReader br = new BufferedReader(new InputStreamReader((new URL(url)).openStream(), StandardCharsets.UTF_8));
             Post video = ytubeScrape.parseVideo(br, "url", "https://www.youtube.com/watch?v=KVGRN7Z7T1A");
             DAO.log(video.toString());
             assertThat(video.get("html_title").toString(), is("[\"Iggy Azalea - Team (Explicit) - YouTube\"]"));
+            assertEquals(postType, video.get("post_type"));
+            assertEquals(postScraper, video.get("post_scraper"));
```

Comment:
Using `assertThat` is preferred here to favor consistency.

Reply:
Are you referring to add the assertions to the other method or to change the ```assertEquals``` to ```assertThat``` or to remove them? @singhpratyush please clarify.

Reply:
Do not use `assertEquals` here. Use `assertThat`.

Reply:
Thanks for the clarification, making changes :+1: